doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -6,131 +6,410 @@ Cite Test Guide
+         to run.
+
+         .. code:: shell
+
+            make
+
+      -  the output will like this:
+
+         .. code:: makefile
+
+            clean: $(suite)         Will Clean the Environment of previous runs.
+            build: $(suite)         Will Build the GeoServer Docker Image for the Environment.
+            test: $(suite)      Will running the Suite test with teamengine.
+            webUI: $(suite)		 Will running the Suite test with teamengine.
+
+      - Choose which test to run, this is an example:
+
+        .. warning::
+
+            The first Docker build may take a long time.
+
+        .. code:: SHELL
+
+           suite=wcs10
+
+        .. note::
+
+           Valid values for the Suite parameter are:
+             * wcs10
+             * wcs11
+             * wfs10
+             * wfs11
+             * wms11
+             * wms13
+
+      - Choose which GeoServer war to test by setting the ``war_url`` environment variable inside the ``Makefile``, ex:
+
+        .. code:: C
+
+          war_url = "https://build.geoserver.org/geoserver/main/geoserver-main-latest-war.zip"
 
-- :ref:`Teamengine Web Application<Teamengine Web Application>`
+      .. note::
+
+        if you don't want to do it inside the ``Makefile`` you have the option of add the variable in the command when you build the docker images.
+
+      -  To clean the local environment.
+
+         .. code:: shell
+
+            make clean suite=<suite-name>
+
+      -  To build the geoserver docker image locally.
+
+         .. code:: shell
+
+            make build suite=<suite-name>
+
+      - Alternative, with the ``war_url`` variable include:
+
+         .. code::
+
+           make build suite=<suite-name> war_url=<url-or-the-geoserver-war-file-desired>
+
+      -  To run the suite test.
+
+         .. code:: shell
+
+            make test suite=<suite-name>
+
+      -  And the last, but no less important run the full automate
+         workflow.
+
+
+         .. code:: shell
+
+            make clean build test suite=<suite-name>
+
+
+Run CITE Test Suites in local pc
+================================
 
 .. note::
 
-   To build every suite test you need to do the next steps:
+   I assume that you have an standalone geoserver running.
+
+.. important::
+
+   Details to have in consideration when you are running the tests:
+
+   - The Default username/password for the teamengine webUI are **teamengine/teamengine**.
 
-  .. code:: shell
+   - the default url for the teamengine webUI is http://localhost:8888/teamengine/
+
+   - The output of the old suite tests, i might not appear in the Result page. So you should click on the link below **detailed old test report**, to get the full report. Ex.
+
+   .. image:: ./image/old-report.png
+
+   .. image:: ./image/full-report.png
+
+   - Add the ip of the geoserver to the docker-compose file, so later you can use that as a host for the url of the tests.
+
+      - Go to the directory ``interactive``.
```

Comment:
This approach makes people modify file in the same repository as GeoServer, while they are working to change the GeoServer source code, to make it pass the CITE tests. We have been over this before, it should not be done, sooner or later someone will commit their specific machine IP and fail.
They can also be on DHCP, with unstable IP addresses.

Best to tell them to just look up their IP address, and enter it i the URLs instead of localhost (they just need to be told to do so, do so here, it's a good place.

Reply:
solved in https://github.com/randomorder/geoserver/pull/3/commits/91da4cbdc686ba5397014ab7887381ee240f8a1b